zml/tokenizer.zig

```diff
@@ -920,15 +929,16 @@ pub fn fromHfJson(allocator: std.mem.Allocator, tokenizer_path: []const u8) !Tok
         else => return error.InvalidFormat,
     };
 
+    const model = objectGet(main_object, .object, "model") orelse return error.InvalidFormat;
     // TODO: remove all panics
```

Comment:
is this comment still valid, I didn't see panics in the diff context?

Reply:
so I had removed most of the .? but had forgotten some.
I remove the last ones and added a more precise TODO wrt EOS/BOS tokens